en/manage-backup.md

```diff
@@ -119,5 +119,12 @@ has regenerated the  `/var/lib/maas/secret` file, you'll need to make sure this
 secret is updated on each rack controller to allow them to re-connect to the
 newly restored region controller.
 
+### Recreating db triggers
```

Comment:
It's not only that. It is possible that the code version from the DB backup is actually different. So you should always run this command to ensure that the schema and the triggers match the currently running code version.